### ISSUE TITLE
Add ability to fully configure map layers when loading via package

### ISSUE THREAD

**Issue by bferguso** [archesproject/arches#8929]
Currently it doesn't appear to be possible to set models.MapLayer attribute values such as `icon`, `addtomap`,`centerx`, etc. when loading map layers as part of a package.

**User Story**
Would like to be able to fully configure the MapLayer when importing them as part of a package. 

**Acceptance Criteria**
-   [ ] map_layer database columns icon, activated, addtomap, centerx, centery, zoom, legend and searchonly can be configured from an Arches package
-   [ ] Unit Tests passed
-   [ ] Documentation

Possible solutions:
1. Use the standard Mapbox style `metadata` JSON element to configure these values in the package.
2. Add this configuration to a separate JSON file in the same directory as the Mapbox style.

**Definition of Done**
map_layer columns are populated with the values configured in the package. Default values would remain the same as the existing default values if not provided in the package.

**Ticket Background**
*   Sponsored by: Self Funded
*   Suggested by: @bferguso
*   Designed by: @bferguso

**Comment by chiatt** [comment]
Hey @bferguso, this looks like a nice improvement. Is this targeting 6.2.x?

**Comment by bferguso** [comment]
Hi @chiatt - I'll probably target 6.2.x for now as we're not ready to move to 7.0.0 just yet, but it looks like the part of `management/commands/packages.py` that would need to be updated hasn't changed since before 6.2.x so the update may merge cleanly into 7.0.x too.

**Comment by chiatt** [comment]
That sounds good. It seems like this can be done without migrations, but one issue that has come up now that we have 7.x available alongside 6.x is that (as far as I can tell) we can't add new migrations to 6.x without breaking the upgrade path to 7.x. If a migration does turn out to be necessary, maybe a setting could be used to store the order - at least as a temporary solution in 6.x.

**Comment by bferguso** [comment]
Right, that makes sense. I think this one can be done without migrations, but #8928 will likely need one. Perhaps I'll need to revisit out 7.0.x upgrade timeline to bring that change in.
